examples/hybrid/advection_agnesi.jl

```diff
@@ -0,0 +1,376 @@
+using Test
```

Comment:
In general @jakebolewski I agree; seems more useful to include a specific terrain warp check via `/test` first; then introduce this example in a separate PR.

Reply:
feel free to remove it